docs/lst_analysis_workflow.rst

```diff
@@ -0,0 +1,124 @@
+    :width: 1080
+    :align: center
+    :alt: LST-1 data analysis pipeline
+
+
+The steps marked with lstchain must be run by the analysers using the corresponding lstchain command (see :ref:`the Analysis Steps <introduction>`) .
+Other tools are described below.
+
+
+lstmcpipe
+---------
+lstmcpipe handles the analysis of MC data on the cluster at La Palma. 
+It produces the trained models and the MC DL2 files required for data analysis.
+Its usage is recommended in order to:
+- avoid common mistakes and pitfalls
+- do not worry about resources management for this part of the analysis which requires specific needs
+- have a reproducible analysis
+- have common analysis production and therefore save computing resources
+
+If you need a specific MC analysis production, you can request one by opening a pull-request on the GitHub repository.
+
+repository: https://github.com/cta-observatory/lstmcpipe
+
+documentation: https://cta-observatory.github.io/lstmcpipe
+
+lstosa
+------
+lstosa handles the analysis of real data on the cluster at La Palma.
+It automatically produces the DL1 files required for data analysis.
+Analysers should not need to use this package directly (it is maintained and handled by LSTOSA team), but use the DL1 files it produces.
+
+repository: https://github.com/cta-observatory/lstosa    
+
+documentation: https://lstosa.readthedocs.io/
+
+
+Files and configuration
+=======================
+
+The DL1 files to use obviously depend on the source you want to analyse.
+Unless you have a good reason, the latest version of the DL1 files should be used.
+
+RF models
+---------
+
+The list of available trained RF models can be found in the lstmcpipe documentation and production list, 
+along with a description of each production:
+https://cta-observatory.github.io/lstmcpipe/productions.html
+
+The RF models are stored in the following directory:
+``/fefs/aswg/data/models/...``
+
+
+Tuning of DL1 files and RF models
+---------------------------------
+
+In case of high NSB in the data, it is possible to tune the DL1 files and the RF models to improve the performance of the analysis.      
+This is done by changing the `config` file of the RF models and producing new DL1 files and training new RF models.
+To produce a config tuned to the data you want to analyse, you may run ``lstchain_tune_nsb`` function that will produce a ``tuned_nsb_config.json`` file.
+
+To request a new production of RF models, you can open a pull-request on the lstmcpipe repository, producing a complete MC config, using:
+
+.. code-block::
+
+    lstchain-dump-config --mc --update-with tuned_nsb_config.json --output-file PATH_TO_OUTPUT_FILE
+
+
+lstchain config
```

Comment:
It would be better to clarify the type of config file here, as we have various different config files (which might require a separate introduction here).

For example, the DL3/IRF config file, as shown in `docs/examples/`, where the `irf_tool_config.json` is the more general config file, which can/should be used for producing DL3 files.

Reply:
I added a DL3/IRF subsection in config to clarify this.